ui/src/layouts/Header/Nav.tsx

```diff
@@ -107,13 +111,20 @@ const NavWrapper = styled.nav`
     $isLoggedIn &&
     `justify-content: flex-end;
   `};
+  .contact {
+    margin-left: var(--smallSpace);
+  }
   .user-nav {
     > * + * {
       margin-left: var(--space);
     }
     .dashboard-nav-link {
-      color: var(--textColor);
       display: block;
+      height: 45px;
+      &:hover {
+        color: var(--button);
+        border-color: var(--purple);
```

Comment:
We shouldn't have to manipulate button styles outside of the button component. That means something is off. All button logic should be handled within the button component so we know that everything is standardized throughout. If there is an outlier style then it means some standards are not being followed properly somewhere within the design

Reply:
you are right i didnt need this styled div class anymore